CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 ## Master (Unreleased)
 
+- Fix an error when using `RSpec/FilePath` and revert to enabled by default. If you have already migrated `RSpec/SpecFilePathSuffix` and `RSpec/SpecFilePathFormat`, no problem. The `RSpec/FilePath` before migration and the `RSpec/SpecFilePathSuffix` and `RSpec/SpecFilePathFormat` at the destination are available respectively. ([@ydah])
```

Comment:
Isn’t it a problem for people had custom configuration for `RSpec/FilePath`, and changed it when upgrading to v2.24.0?

Reply:
Indeed it is. Explicitly disabled `RSpec/FilePath`. How about this?